content/en/account_management/authen_mapping/_index.md

```diff
@@ -0,0 +1,487 @@
+
+{{< tabs >}}
+{{% tab "Example" %}}
+
+{{< code-block lang="bash" filename="roles.sh" >}}
+{�
+  "type": "authnmapping",
+  "data": {�
+    "attributes": {�
+      "role_uuid": "00000000-0000-0000-0000-000000000000",
+      "attribute_key": "string",
+      "attribute_key": "string"
+    }
+  }
+}
+{{< /code-block >}}
+
+Replace the `<YOUR_DATADOG_API_KEY>` and `<YOUR_DATADOG_APPLICATION_KEY>` placeholder with the corresponding [API and application keys for your organization][1].
+
+[1]: https://app.datadoghq.com/account/settings#api
+{{% /tab %}}
+{{% tab "Response" %}}
+
+{{< code-block lang="json" filename="response.json" disable_copy="true" >}}
+{
+  "data": {
+    "attributes": {
+      "created_at": "string",
+      "created_by": "string",
+      "uuid": "10000000-abcd-efgh-ijkl-000000000000",
+      "role_uuid": "00000000-0000-0000-0000-000000000000",
+      "saml_assertion_attribute_id": 0
+    },
+    "type": "string",
+    "id": "string",
+    "relationships": {
+      "saml_assertion_attributes": {
+        "data": {
+          "id": 0,
+          "type": "saml_assertion_attributes"
+        }
+      },
+      "roles": {
+        "data": {
+          "id": "string",
+          "type": "roles"
+        }
+      }
+    }
+  },
+  "included": [
+    {
+      "data": {
+        "id": "string",
+        "type": "roles",
+        "attributes": {
+          "created_at": "string",
+          "modified_at": "string",
+          "uuid": "00000000-0000-0000-0000-000000000000",
+          "name": "string"
+        },
+        "relationships": {
+          "data": [
+            {
+              "id": "string",
+              "type": "permissions"
+            }
+          ]
+        }
+      }
+    },
+    {
+      "data": {
+        "id": 0,
+        "type": "authn_mappings",
+        "attributes": {
+          "id": 6,
+          "attribute_key": "string",
+          "attribute_value": "string"
+        }
+      }
+    }
+  ]
+}
+{{< /code-block >}}
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Get all AuthN Mappings
+
+Returns a list of AuthN Mappings
+
+| Method | Endpoint path            | Required payload |
+|--------|-------------------------|------------------|
+| `GET`  | `/v2/authn_mappings` | optional queries       |
+
+##### ARGUMENTS
+
+* **`sort`** [*optional*, *default*=**yes**]:
+  Sort attribute and direction - `yes` sorts in descending order, `no` sorts in ascending order.
+* **`page_number`** [*optional*, *default*=**0**, *minimum*=**0**]:
+  The page of results to return. 
+* **`page_size`** [*optional*, *default*=**10**]:
+  The number of results to return on each page.
+* **`filter`** [*optional*, no default]:
+  Filter by tags as strings. For example, `Billing Users`.
```

Comment:
1. get_all query params are as follows
        1. Sort is the name of the field you want to sort by, if you want it to be ascending, you pass it in as a negative e.g "-created_at", default is ascending, e.g "created_at"
        2. page[number], default is 0, page number that a user requests
        3. page[size], default is 10, number of entries per page that a user requests
        4. filter, default is None, filter string that a user passes in